client/my-sites/stats/stats-module-utm/stats-module-utm-summary.jsx

```diff
@@ -0,0 +1,204 @@
+import { useQuery } from '@tanstack/react-query';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import wpcom from 'calypso/lib/wp';
+import StatsModuleDataQuery from '../stats-module/stats-module-data-query';
+import statsStrings from '../stats-strings';
+import UTMDropdown from './stats-module-utm-dropdown';
+
+const OPTION_KEYS = {
+	SOURCE_MEDIUM: 'utm_source,utm_medium',
+	CAMPAIGN_SOURCE_MEDIUM: 'utm_campaign,utm_source,utm_medium',
+	SOURCE: 'utm_source',
+	MEDIUM: 'utm_medium',
+	CAMPAIGN: 'utm_campaign',
+};
+
+function StatsModuleUTMSummary( { siteId, period, postId, query, summary, className } ) {
```

Comment:
This looks like an exact copy of the other module. Wouldn't it be easier to import either the other UTM module or the `stats module` (used by the other components, like top posts or referrers)?
This approach would create a third version of the component. What's the reason behind it?

Reply:
The main reasoning is because I'm using a different query hook. The goals here were:

1. To get the page working with the period controls.
2. To avoid firing off 100+ network requests.
3. To test Tanstack without Redux in an isolated case.

The idea was to test it out, then check with the team for input on the approach. Once we have a consensus for the API calls (make multiple requests vs update the API) we could iterate and consolidate.